Nos/Views/ProfileEdit/ProfileEditWizard.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ProfileEditWizard.swift
+//  Nos
+//
+//  Created by Martin Dutra on 6/3/24.
+//
+
+import SwiftUI
+
+protocol ProfileEditWizard: View { }
+
+extension ProfileEditWizard {
+    // swiftlint:disable identifier_name
+    func WizardNavigationStack<Root: View>(@ViewBuilder root: () -> Root) -> some View {
```

Comment:
Why not just make this a `View` instead of a `func`?

Reply:
The idea there was to provide a way to reuse code in the ProfileEdit views without exposing them to other parts of the codebase because that code is very specific to these views. It's like having a namespace, if you declare your struct to inherit ProfileEditWizard instead of View, you can access these funcs otherwise, this code never bothers you.

I'm open to discuss this anyway, its a novel idea that if it looks too weird, we can switch to something else.

Reply:
Ah, that makes sense. I think I'd just go with Views to stick with the conventional way. For one, this solves the naming issue (capitalized names are usually types). I suppose I'm generally fine with `func`s that produce views as long as they're capitalized the way `func`s normally are.

As for namespacing: I'm not too concerned about that since our codebase is pretty small. But if we wanted to, I think we could declare the views inside of ProfileEditWizard, maybe even inside of an extension. So something like this should work:

```
extension ProfileEditWizard {
    struct WizardNavigationStack: View { ... }
}
```

The way I generally like to do this, though, would be to create a Components folder that would include things like WizardNavigationStack and ActionButton and ActivityView and such. And then those components would be available to any other view.